addons/website_mass_mailing/models/res_company.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def _get_social_media_links(self):
+        social_media_links = super()._get_social_media_links()
+        website_id = self.env['website'].get_current_website()
+        social_media_links.update({
+            'social_facebook': website_id.social_facebook or self.social_facebook,
```

Comment:
fallback should use the value from super() and not re-do the same.

If a third module change it in an override, you will loose the change.